### PR TITLE
Adding all HRI documentation files/assets for 1st time

### PR DESCRIPTION
**Pull request by aramcodz**
all files here copied From (internal IBM WH) Release `alvearie-release-v0.3.0`
This is the Alvearie v0.3.0 version of the Documentation

Signed-off-by: Aram Openden <Aram.Openden1@ibm.com>

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
+@import url("https://fonts.googleapis.com/css?family=Arvo:400,700,400italic");
```
Comment (David-N-Perkins) [inline assets/css/style.css:1]: I think this file is automatically downloaded by Jekyll and should be included in the repo.  It's not in our internal docs repo.

File: assets/js/scale.fix.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+fixScale = function(doc) {
```
Comment (David-N-Perkins) [inline assets/js/scale.fix.js:1]: I think this file is automatically downloaded by Jekyll and should be included in the repo. It's not in our internal docs repo.

**Review by David-N-Perkins — COMMENTED** [review]
There's several images that shouldn't be included. Here are a few, but stopped looking at them:
- `assets/img/activity_tracker_event_streams.png`
- `assets/img/architecture-with-validation-core.png`
- ` assets/img/create_topic_msg_retention.png`
- `create_topic_name.png`
- `create_topic_num_partitions.png`

**Comment by aramcodz** [comment]
> There's several images that shouldn't be included. Here are a few, but stopped looking at them:
> 
> * `assets/img/activity_tracker_event_streams.png`
> * `assets/img/architecture-with-validation-core.png`
> * ` assets/img/create_topic_msg_retention.png`
> * `create_topic_name.png`
> * `create_topic_num_partitions.png`

Q. How did you come up with this list of image files that should Not be included?

**Comment by aramcodz** [comment]
> There's several images that shouldn't be included. Here are a few, but stopped looking at them:
> 
> * `assets/img/activity_tracker_event_streams.png`
> * `assets/img/architecture-with-validation-core.png`
> * ` assets/img/create_topic_msg_retention.png`
> * `create_topic_name.png`
> * `create_topic_num_partitions.png`

All extraneous images in assets/img have now been removed.

File: assets/css/style.css (hunk 1 above)
Comment (aramcodz) [reply assets/css/style.css]: Yup. I have Removed this file now.

File: assets/js/scale.fix.js (hunk 2 above)
Comment (aramcodz) [reply assets/js/scale.fix.js]: Same: you are corred, @David-N-Perkins. File now removed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+* Prerequisites:
+  * Requires Ruby 2.6.3 or higher
+    * Check your Ruby version at cmd line: `$ ruby --version`
+    * To upgrade your Ruby version go, follow [instructions here to install & run rvm ](https://codingpad.maryspad.com/2017/04/29/update-mac-os-x-to-the-current-version-of-ruby).
+    * Your rvm command will be something like: `$ rvm install ruby-2.6.3`   
+    * You may also need to run `gem update --system`.
+    * If you are successful with that, continue onto running the bundler in the next step
+
+* Run the bundler: `bundle install`
+* Make the template think it's running in github pages by setting the env var: `export PAGES_REPO_NWO=wffh-hri/docs`
```
[the remark this answers is not in the document]
Comment (aramcodz) [reply README.md]: Good Catch - This has now been Fixed - see latest.

**Review by nathandlee — APPROVED** [review]

**Review by David-N-Perkins — COMMENTED** [review]
I think these need to be removed too:
assets/images/arrow-down.png
assets/images/octocat-small.png
assets/img/iam_data_integrator_policies.png

**Comment by aramcodz** [comment]
> I think these need to be removed too:
> assets/images/arrow-down.png
> assets/images/octocat-small.png
> assets/img/iam_data_integrator_policies.png

Yup. I missed those three files. They have now been removed.

**Review by David-N-Perkins — APPROVED** [review]

**Review by FRicci618 — APPROVED** [review]

**Pull request merged**